recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge rc_cupy
```

Comment:
This is the channel and label needed by Conda to specify this package (once it is published).